wordpress-mobile/WordPress-FluxC-Android
# ThemeStore: Add actions to install themes on Jetpack sites

Theme activation for both Jetpack and WP.com was added in #552. While any WP.com theme can be activated on WP.com sites without issue, Jetpack sites require the theme to be installed first. This PR adds the necessary install actions and updates the example app to demo and help with testing.

To test:
* Go to the site's wp-admin and check which themes are currently installed
* In the example app...sign into an account with a Jetpack site linked
* Go to the Theme's screen
* Type a theme slug (ie `edin`, `twentysixteen`, etc...) for a theme that isn't already installed into the EditText
* Click the `Install WP.com theme on Jetpack site` button and check the log output to make sure the installation was successful
* Now try again with the same theme and make sure the log shows `The theme is already installed`

cc @aforcier, would you mind taking a look at this one?
